library/Zend/InputFilter/InputInterface.php

```diff
@@ -63,6 +70,12 @@ public function setValidatorChain(ValidatorChain $validatorChain);
     public function setValue($value);
 
     /**
+     * @param  mixed $value
+     * @return self
+     */
+    public function setFallbackValue($value);
```

Comment:
We cannot add methods to existing interfaces (causes a BC break). We *can* create a new interface that defines this method, and update `Input` to use that interface (instead of only defining it within the concrete class).